Makefile

```diff
@@ -438,6 +451,19 @@ ifneq ($(HIP_LIB_DIR),)
   BACKENDS_MAKE += $(HIP_BACKENDS)
 endif
 
+# SYCL Backends
```

Comment:
Don't forget to add the magic decoder ring for user facing make options to the README